languages/erlang/docs/implementing-a-concept-exercise.md

```diff
@@ -1,5 +1,71 @@
 # How to implement an Erlang concept exercise
 
-TODO: describe how to implement a concept exercise for the Erlang track. For inspiration, check out the [C# version of this file][csharp-implementing].
+This document describes how to implement a concept exercise for the Erlang track.
+As this document is generic, the following placeholders are used:
 
-[csharp-implementing]: ../../csharp/docs/implementing-a-concept-exercise.md
+- `<SLUG>`: the name of the exercise in kebab-case (e.g. `anonymous-functions`).
+- `<NAME>`: the name of the exercise in snake_case (e.g. `anonymous_functions`).
+
+Before implementing the exercise, please make sure you have a good understanding of what the exercise should be teaching (and what not).
+This information can be found in the exercise's GitHub issue.
+Having done this, please read the [Erlang concept exercises introduction][concept-exercises].
+
+```plain
+languages
+└── erlang
+    └── exercises
+        └── concept
+            └── <SLUG>
+                ├── .docs
+                |   ├── instructions.md
+                |   ├── introduction.md
+                |   ├── hints.md
+                |   └── after.md (optional)
```

Comment:
The after.md is no longer optional, and we've since added a source.md file (which is optional in certain cases):

```suggestion
                │   ├── instructions.md
                │   ├── introduction.md
                │   ├── hints.md
                │   ├── after.md
                │   └── source.md (required if there are third-party sources)    
```